modules/TableFactory/src/main/java/com/percussion/tablefactory/PSJdbcStatementFactory.java

```diff
@@ -1567,6 +1571,49 @@ private static PSJdbcSqlStatement getAddComponentStatement(String tableName,
       return new PSJdbcSqlStatement(buf.toString());
    }
 
+   /**
+    * Generates an alter column statement to alter column defination as defined.
```

Comment:
Please turn on warnings / spell check on in IDE

Reply:
is there.. but not taking advantage :)